ui-admin/src/study/kits/KitEnrolleeSelection.tsx

```diff
@@ -75,29 +81,20 @@ export default function KitEnrolleeSelection({ studyEnvContext }: { studyEnvCont
   const numSelected = Object.keys(rowSelection).length
   const enableActionButtons = numSelected > 0
 
-  const studyColumns: ColumnDef<EnrolleeRow, string | boolean>[] = currentEnv.configuredSurveys.length === 0 ? [] :
-    currentEnv.configuredSurveys
-      .map(configuredSurvey => {
-        const survey = configuredSurvey.survey
-        return {
-          id: survey.stableId,
-          header: survey.name,
-          accessorFn: enrollee => enrollee.taskCompletionStatus[survey.stableId],
-          meta: {
-            columnType: 'boolean',
-            filterOptions: [
-              { value: true, label: 'Completed' },
-              { value: false, label: 'Not Completed' }
-            ]
-          },
-          filterFn: 'equals',
-          cell: ({ row: { original: enrollee } }) => enrollee.taskCompletionStatus[survey.stableId]
-            ? <FontAwesomeIcon icon={faCheck}/>
-            : ''
-        }
-      })
+  const requiredSurveys = currentEnv.configuredSurveys.filter(survey => survey.required)
```

Comment:
This is such a 🤦‍♂️ moment. I don't know what I was thinking `required` meant in the survey model. Maybe that the survey is required for the user's data to be included in the dataset? I guess I'm still wondering if we want different requirements for being included in the dataset and kit eligibility.

Reply:
Yeah, I think eventually we'll need to have to support an explicit "eligibleForKit" rule string, since folks will want to filter on state geography and lots of other stuff.  but for now it's a nice coincidence that kit eligibility == finished required surveys